source/operations/install-deploy-manage/deploy-minio-tenant.rst

```diff
@@ -563,14 +538,236 @@ the Kubernetes cluster.
 
-You can also use this procedure to generate a Kubernetes ``YAML``-formatted resource document for further customization.
+To deploy a tenant from the command line, complete the following steps:
+
+:ref:`create-tenant-cli-determine-settings-required-options`
+
+:ref:`create-tenant-cli-determine-additional-options`
+
+:ref:`create-tenant-cli-enter-command`
+
+:ref:`create-tenant-cli-record-access-info`
+
+:ref:`create-tenant-cli-access-tenant-console`
+
+:ref:`create-tenant-cli-forward-ports`
+
+.. _create-tenant-cli-determine-settings-required-options:
+
+1) Determine Settings for Required Settings
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The :mc:`kubectl minio tenant create` command requires several configuration settings.
+Determine the values for all required flags.
+
+For example, the following command creates a new tenant with 
+  
+- the name ``minio_tenant``
+- total capacity of 16 Tebibytes
+- four servers
+- 16 volumes
+- the ``minio`` namespace.
+
+.. code-block:: shell
+   :class: copyable
+
+   kubectl minio tenant create minio_tenant --capacity 16Ti --servers 4 --volumes 16 --namespace minio
+
+The command requires values for each of the following:
+
+- :mc:`~kubectl minio tenant create TENANT_NAME`
+    
+  The name to use for the new tenant.
+
+- :mc:`~kubectl minio tenant create --capacity`
+
+  The total raw storage size for the Tenant across all volumes. 
+  Specify both the total storage size *and* the :guilabel:`Unit` of that storage. 
+  All storage units are in SI values, e.g. :math:`Gi = GiB = 1024^3` bytes.
+     
+  For example, 16 Ti for 16 Tebibytes.
+
+- :mc:`~kubectl minio tenant create --servers`
+
+  The total number of MinIO server pods to deploy in the Tenant.
+    
+  The Operator by default uses pod anti-affinity, such that the Kubernetes cluster *must* have at least one worker node per MinIO server pod.
+
+- :mc:`~kubectl minio tenant create --volumes`
+
+  The total number of storage volumes (Persistent Volume Claims). 
+  The Operator generates an equal number of PVC *plus two* for supporting Tenant services (Metrics and Log Search).
+       
+  The total number of persistent volume claims (``PVC``) per server is determined by dividing the number of volumes by the number of servers.
+  The storage available for each ``PVC`` is determined by dividing the capacity by the number of volumes. 
+
+  The generated claims have pod selectors so that claims are only made for volumes attached to node running the pod.
+
+  If the number of volumes exceeds the numnber of persistent volumes available on the cluster, ``MinIO`` hangs until the number of persistent volumes are available.
+  
+- :mc:`~kubectl minio tenant create --namespace`
+
+  Each MinIO tenant requires its own ``namespace``.
+
+  When not specified, the MinIO Operator defaults to use ``minio``.
+  To specify a different namespace, use the :mc:`~kubectl minio tenant create --namespace` flag.
```

Comment:
Since we're saying these are required, its odd to talk about a default value at all. Maybe

"Omit this value to use the default namespace ``minio``."

Or drop this section entirely and go straight to 622

Reply:
Did a few updates to maybe address this.